library/ssl_tls12_server.c

```diff
@@ -1131,7 +1131,7 @@ static int ssl_parse_client_hello(mbedtls_ssl_context *ssl)
      * ClientHello layer:
      *     0  .   1   protocol version
      *     2  .  33   random bytes (starting with 4 bytes of Unix time)
-     *    34  .  35   session id length (1 byte)
+     *    34  .  34   session id length (1 byte, x = session id length)
      *    35  . 34+x  session id
```

Comment:
It might be more logical to place the `x = ...` on this line, i.e.

```
     *    34  .  34   session id length
     *    35  . 34+x  session id, where x = session id length from byte 34
```

Reply:
Thanks.  I have changed the code.